invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -538,15 +528,15 @@
                 "type": "string",
                 "description": "A reference string."
               },
-              "identifier": {"type": "string"},
-              "scheme": {"type": "string"}
+              "identifier": {"$ref": "#/definitions/identifier"},
+              "scheme": {"$ref": "#/definitions/scheme"}
             }
           }
         }
       }
     },
 
-    "metaext": {
+    "ext": {
       "type": "object",
       "description": "Configured additional metadata",
       "propertyNames": {
```

Comment:
Question: ``"pattern": "^[A-Za-z][A-Za-z0-9_]*:[A-Za-z0-9_]+$"`` has to be changed as well? Do you keep a list of things to change after merging somewhere?

Reply:
Ah, yes, didn't fix the internal part of it. Will do when I arrive to the field itself.

Reply:
Didn't fix the field inside. Waiting to actually fix the metadata extensions since also loading dumping needs fix after all the changes.